kafka-client/src/main/java/dev/responsive/kafka/api/ResponsiveDriver.java

```diff
@@ -152,6 +160,44 @@ public KeyValueBytesStoreSupplier globalKv(final String name) {
     return new ResponsiveGlobalKeyValueBytesStoreSupplier(client, name, executor);
   }
 
+  public <K, V> StoreBuilder<TimestampedKeyValueStore<K, V>> timestampedKeyValueStoreBuilder(
+      final String name,
+      final Serde<K> keySerde,
+      final Serde<V> valueSerde
+  ) {
+    return new TimestampedKeyValueStoreBuilder<>(
+        timestampedKv(name),
+        keySerde,
+        valueSerde,
+        Time.SYSTEM) {
+      private final Logger log =
+          new LogContext(String.format("store-builder [%s]", name)).logger(ResponsiveDriver.class);
+
+      @Override
+      public StoreBuilder<TimestampedKeyValueStore<K, V>> withLoggingEnabled(
```

Comment:
I still think we should throw an exception here instead of silently overriding - otherwise we risk accidentally truncating/deleting a topic that they don't want deleted, which would be something that (even if it's their "fault" for not understanding our code) would be something really hard to recover trust from.